elastic/kibana
# [ResponseOps][Connectors] add the "service message" to the message generated for errors

Resolves https://github.com/elastic/kibana/issues/187288

## Summary

When a connector fails we log the error message. In this PR I updated the log message to include the error message followed by the serviceMessage if it's populated. This change will help provide more detailed info in the log messages when a connector fails.


### Checklist



### To verify

- Create a connector and update the executor code to make it fail
- Create rule with your connector
- Verify that the log message includes more details about the error instead of something like the following example for email connectors: `Action '[email connector name]' failed: error sending email"`
